net-libs/nodejs/nodejs-20.6.1-r1.ebuild

```
@@ -41,8 +41,7 @@ RDEPEND=">=app-arch/brotli-1.0.9:=
 	sys-libs/zlib
 	corepack? ( !sys-apps/yarn )
 	system-icu? ( >=dev-libs/icu-67:= )
-	system-ssl? ( >=dev-libs/openssl-1.1.1:0= )
-	sys-devel/gcc:*"
+	system-ssl? ( >=dev-libs/openssl-1.1.1:0= )"
 BDEPEND="${PYTHON_DEPS}
 	dev-util/ninja
```

This needs to be changed to

```suggestion
	app-alternatives/ninja
```